### PR TITLE
Added more documentation datasets

### PR DESCRIPTION
**Pull request by afranklin**
Adding datasets to documentation so its easier for users to follow along.

### CODE REVIEW

File: userguide/recommender/README.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ interaction data and use that model to make recommendations.
 
 Creating a recommender model typically requires a data set to use for
 training the model, with columns that contain the user IDs, the item
-IDs, and (optionally) the ratings. For this example, we use the [MovieLens
- dataset](https://grouplens.org/datasets/movielens/).
+IDs, and (optionally) the ratings. For this example, we use the [MovieLense
```
Comment (znation) [inline userguide/recommender/README.md]: Should be `MovieLens 20M` (no e)?

File: userguide/supervised-learning/random_forest_classifier.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ forests.
 
 ##### Introductory Example
 
-In this example, we will use the [Mushrooms dataset](https://archive.ics.uci.edu/ml/datasets/mushroom).
+In this example, we will use the [Mushrooms dataset](https://archive.ics.uci.edu/ml/datasets/mushroom).decision_tree_regression.md
```
Comment (znation) [inline userguide/supervised-learning/random_forest_classifier.md]: This looks like a typo.

File: userguide/supervised-learning/random_forest_regression.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ forests, all the base models are constructed independently using a
 
 ##### Introductory Example
 
-In this example, we will use the [Mushrooms dataset](https://archive.ics.uci.edu/ml/datasets/mushroom).
+In this example, we will use the [Mushrooms dataset](https://archive.ics.uci.edu/ml/datasets/mushroom).decision_tree_regression.md
```
Comment (znation) [inline userguide/supervised-learning/random_forest_regression.md]: This looks like a typo.

File: userguide/supervised-learning/random_forest_classifier.md (hunk 2 above)
Comment (afranklin) [reply userguide/supervised-learning/random_forest_classifier.md]: Thanks for the attention to detail - fixed!

**Comment by afranklin** [comment]
Thanks @znation , I fixed both issues you raised.

File: userguide/supervised-learning/random_forest_classifier.md (hunk 2 above)
Comment (znation) [reply userguide/supervised-learning/random_forest_classifier.md]: I still see the typo on this file.

**Comment by arashsa2022** [comment]
is this https://static.turi.com/datasets/bitcoin/market-price.csv data set available? I use a code related to 2016 that used this data set

**Pull request merged**